Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -695,9 +695,29 @@ def test_log
         system "git", "commit", "--allow-empty", "-m", "This is a test commit"
       end
     end
+
+    core_tap = CoreTap.new
```

Comment:
@MikeMcQuaid We've discussed the idea of updating the existing `test/testball.rb` in lieu of relying on a `testball` formula being written into the core tap, so this update is on my to-do list. 😄 

Until I figure out how to do that, would it be OK to go ahead and submit a separate PR for this refactoring?

There are 18 declarations of `formula_file` in the file (including the one in my PR). If we pull these into a utility method, would the method live in the same file?